fetch/api/request/request-cache-default-conditional.html

```diff
@@ -14,15 +14,15 @@
     <script>
     var tests = [
       {
-        name: 'RequestCache "default" mode with an If-Modified-Since header is treated similarly to "no-store"',
+        name: 'RequestCache "default" mode with an If-Modified-Since header (following a request without headers) is treated similarly to "no-store"',
```

Comment:
Fair enough. I've changed the text from "without headers" to "without additional headers"